cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -212,6 +212,95 @@ tgc.signing.key=szxK-5_eJjs-aUj-64MpUZ-GPPzGLhYPLGl0wrYjYNVAGva2P0lLe6UGKGM7k8dW
 # cas.webflow.keypassword=changeit
 
 ##
```

Comment:
This is certainly the right to think about the plist of the `cas.properties` file.